src/hu_nmt/data_augmentator/augmentators/subject_object_augmentator.py

```diff
@@ -19,24 +19,24 @@
 
 class SubjectObjectAugmentator(AugmentatorBase):
 
-    def __init__(self, eng_graphs: Optional[List[DependencyGraphWrapper]],
-                 hun_graphs: Optional[List[DependencyGraphWrapper]],
+    def __init__(self, src_graphs: Optional[List[DependencyGraphWrapper]],
+                 tgt_graphs: Optional[List[DependencyGraphWrapper]],
```

Comment:
Don't you need ` = None` here?
```suggestion
    def __init__(self, src_graphs: Optional[List[DependencyGraphWrapper]] = None,
                 tgt_graphs: Optional[List[DependencyGraphWrapper]] = None,
```

Reply:
You are right, it makes sense having a default `None` there. It used to be envoked explicitly with Nones [like this](https://github.com/attilanagy234/syntax-augmentation-nmt/blob/main/src/hu_nmt/data_augmentator/entrypoints/run_subj_obj_augmentation.py#L46). 

I'll make the changes.